components/hab/src/cli.rs

```diff
@@ -1198,10 +1211,10 @@ fn sub_sup_run(_feature_flags: FeatureFlag) -> App<'static, 'static> {
                               (Note: This option is explicitly undocumented and for testing purposes only. Do not use it in a production system. Use the corresponding environment variable instead.) \
                               (ex: hab sup run --ring-key 'SYM-SEC-1 foo-20181113185935 GCrBOW6CCN75LMl0j2V5QqQ6nNzWm6and9hkKBSUFPI=')")
                             (@arg CHANNEL: --channel +takes_value default_value[stable]
-                             "Receive Supervisor updates from the specified release channel")
+                             "Receive updates from the specified release channel")
```

Comment:
Yeah, the channel flag is very confusing. As I just recently discovered it does not dictate where the supervisor is initially pulled from.

The reason I was removing it was to keep the `clap_app` version of the cli consistent with the `structopt` version. In the `structopt` version I switched to using this [shared struct](https://github.com/habitat-sh/habitat/blob/4b1e61f6b36abcbb5032880017d5644d25f1df0c/components/hab/src/cli/hab/svc.rs#L90) for `hab sup run` and `hab svc load`.

I figured removing the word "Supervisor" would be ok because technically it is any update to the Supervisor or services. Maybe removing the word "Supervisor" is even more clear? I can add it back if you prefer and just duplicate the `channel` field.

Reply:
Should users update Supervisors and the services from the same release channel? 
My understanding is that they are separate things, each service run on a supervisor has its own release channel to update itself from, while the supervisor itself has its own release channel to update from.